docs/usage/zip.md

```diff
@@ -0,0 +1,47 @@
+Zip
+========
+
+Allows you to create a temporary zip file of Box files and folders and download them.
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+
+- [Create a Zip File](#create-a-zip-file)
+- [Download a Zip File](#download-a-zip-file)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+Create a Zip File
+-----------------------------
+
+Calling [`client.create_zip(name, items)`][create_zip] will let you create a new zip file with the specified name and with the specified items and will return a `dict` with the download and status link. This file does not show up in your Box account, but will be temporarily available for download.
```

Comment:
```suggestion
Calling [`client.create_zip(name, items)`][create_zip] will let you create a new zip file with the specified file name and containing the specified files and/or folders.  It will return a `dict` containing the download link and status link. This zip file will not show up in your Box account, but will be temporarily available for download.
```